<ui version="4.0" >
 <class>StatisticClass</class>
 <widget class="QWidget" name="StatisticClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1253</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Statistic</string>
  </property>
  <widget class="QListWidget" name="listWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>50</y>
     <width>211</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="tableComboBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="socialComboBox" >
   <property name="geometry" >
    <rect>
     <x>270</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="percentLabel" >
   <property name="geometry" >
    <rect>
     <x>500</x>
     <y>330</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>SimSun</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text" >
    <string>0%</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget" >
   <property name="geometry" >
    <rect>
     <x>270</x>
     <y>50</y>
     <width>681</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton" >
   <property name="geometry" >
    <rect>
     <x>500</x>
     <y>10</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>添加>></string>
   </property>
  </widget>
  <widget class="QListWidget" name="percentListWidget" >
   <property name="geometry" >
    <rect>
     <x>1070</x>
     <y>80</y>
     <width>121</width>
     <height>521</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="statButton" >
   <property name="geometry" >
    <rect>
     <x>370</x>
     <y>10</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>SimSun</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text" >
    <string>统计</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectAllButton" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>460</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>全选</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectNoneButton" >
   <property name="geometry" >
    <rect>
     <x>110</x>
     <y>460</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>全不选</string>
   </property>
  </widget>
  <widget class="QLabel" name="itemLabel" >
   <property name="geometry" >
    <rect>
     <x>980</x>
     <y>50</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton" >
   <property name="geometry" >
    <rect>
     <x>580</x>
     <y>10</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>&lt;&lt; 清空</string>
   </property>
  </widget>
  <widget class="QLabel" name="showLabel" >
   <property name="geometry" >
    <rect>
     <x>850</x>
     <y>610</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>271</x>
     <y>460</y>
     <width>281</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2" >
    <item>
     <widget class="QLabel" name="label_6" >
      <property name="text" >
       <string>设置文件名：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="fileNameLineEdit" />
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>560</x>
     <y>460</y>
     <width>446</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3" >
    <item>
     <widget class="QLabel" name="label_5" >
      <property name="text" >
       <string>选择导出目录</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="openPushButton" >
      <property name="text" >
       <string>..浏览</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="dirLineEdit" />
    </item>
    <item>
     <widget class="QPushButton" name="exportButton" >
      <property name="text" >
       <string>导出</string>
      </property>
      <property name="iconSize" >
       <size>
        <width>13</width>
        <height>13</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>exportButton</zorder>
   <zorder>label_5</zorder>
   <zorder>openPushButton</zorder>
   <zorder>dirLineEdit</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
